### PR TITLE
Fix a 'generate kube' bug on ctrs with named volumes

### PR DESCRIPTION
**Pull request by mheon**
Also, restore some errors that were being ignored.

Comment out an error line because I'm not actually sure what it's doing and why it's there, but we were hitting it on occasion.

### CODE REVIEW

**Comment by baude** [comment]
lgtm

**Comment by TomSweeneyRedHat** [comment]
I'm a little leery about the return removal, but if @baude is hip with that, then the rest LGTM.

File: libpod/kube.go [hunk 1]
```diff
@@ -220,12 +220,12 @@ func containerToV1Container(c *Container) (v1.Container, error) {
 			return kubeContainer, err
 		}
 		kubeContainer.VolumeMounts = volumes
-		return kubeContainer, errors.Wrapf(ErrNotImplemented, "volume names")
+		//return kubeContainer, errors.Wrapf(ErrNotImplemented, "volume names")
```
Comment (rhatdan) [inline libpod/kube.go]: If we are ignoring the error then remove this line, otherwise why are we commenting it out?

**Comment by baude** [comment]
remove the comment per @rhatdan and we can merge

**Comment by mheon** [comment]
Repushed with comment gone

**Pull request merged**